src/toolkits/style_transfer/style_transfer.hpp

```diff
@@ -114,6 +121,8 @@ class EXPORT style_transfer : public ml_model_base {
 
   std::unique_ptr<table_printer> training_table_printer_;
 
+  gl_sarray convert_types_to_sarray(const variant_type& data);
```

Comment:
I believe this could be a static method (doesn't depend on any instance variables)